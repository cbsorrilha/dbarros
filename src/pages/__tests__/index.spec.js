import React from "react"
import renderer from "react-test-renderer"
import BlogIndex from '../index'


describe('Snapshot testing the BlogIndex', () => {
    it("renders correctly", () => {
        const props = {
            data: {
                site: {
                    siteMetadata:{
                        title: "Tchurunarublazer",
                        profilePic: "img",
                        social: {
                            github:  {
                                username: 'cbsorrilha',
                                link: 'link'
                            }
                        }
                    }
                }
            },
            location: {}
        }
        const tree = renderer.create(<BlogIndex {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
