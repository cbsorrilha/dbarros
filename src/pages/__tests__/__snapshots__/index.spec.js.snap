// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the BlogIndex renders correctly 1`] = `
Array [
  <header
    className="sc-AykKC fLWTSL"
  >
    <h1
      className="sc-AykKE jhfoAy"
    >
      <a
        className="sc-AykKD ipEYTO"
        href="/"
      >
        <img
          alt="dbarros logo"
          src="test-file-stub"
        />
      </a>
    </h1>
    <nav
      className="sc-fzXfLQ bimKsw"
    >
      <a
        href="/blog"
      >
        Blog
      </a>
      <a
        href="/"
      >
        Sobre
      </a>
    </nav>
  </header>,
  <main
    className="sc-fzXfLR bMWpZY"
  >
    <div
      className="sc-fzXfLS eLISqO"
    >
      <div
        className="sc-fzXfLT lQQZe"
      >
        <h2>
          Olá, eu sou o Cesar
        </h2>
        <p>
          Sou desenvolvedor web desde 2013 e meu sonho é transformar a Baixada Fluminense (RJ) no novo Vale do Silício™.
        </p>
        <p>
          Desenvolvo no back-end com NodeJS e um pouquinho de Python, no front-end com React, Vue e o que mais vier e sei até um pouquinho de infra com docker e kubernetes.
        </p>
        <p>
          Também me meto escrever na internet.
        </p>
        <p>
          Quer saber mais? Fala comigo.
        </p>
        <div
          className="sc-fzXfLU fCIfcK"
        />
      </div>
      <img
        alt="Eu"
        title="Eu sou o da direita"
      />
    </div>
  </main>,
  <footer
    className="sc-AykKF gWljpi"
  >
    <div
      className="sc-fzXfLO iqPeVM"
    >
      <pre>
        <span
          className="sc-AykKG fUcpJu"
        >
          if
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          (
        </span>
        <span
          className="sc-AykKI iwVWIw"
        >
          true
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          )
        </span>
        <span
          className="sc-AykKJ cteSdM"
        >
          {
        </span>
         
        <span
          className="sc-AykKK gtdKVi"
        >
          loveToCode
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          (
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          )
        </span>
        <span
          className="sc-AykKJ cteSdM"
        >
          ;
        </span>
         
        <span
          className="sc-AykKJ cteSdM"
        >
          }
        </span>
      </pre>
    </div>
    <div
      className="sc-fzXfLP cKvoPT"
    >
      © 
      2020
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
      >
        Gatsby
      </a>
       
      and ♥
    </div>
  </footer>,
]
`;
